recipes/automake/all/conanfile.py

```diff
@@ -0,0 +1,120 @@
+    license = ("GPL-2.0-or-later", "GPL-3.0-or-later")
+
+    settings = "os_build", "arch_build"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
+
+    @property
+    def _version_major_minor(self):
+        [major, minor, _] = self.version.split(".", 2)
+        return '{}.{}'.format(major, minor)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
+
+    def requirements(self):
+        self.requires("autoconf/2.69")
+        # automake requires perl-Thread-Queue package
+
+    def build_requirements(self):
+        if self.settings.os_build == "Windows" and "CONAN_BASH_PATH" not in os.environ \
+                and tools.os_info.detect_windows_subsystem() != "msys2":
+            self.build_requires("msys2/20190524")
+
+    @property
+    def _datarootdir(self):
+        return os.path.join(self.package_folder, "bin", "share")
+
+    @property
+    def _automake_perllibdir(self):
+        return os.path.join(self._datarootdir, "automake-{}".format(self._version_major_minor))
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        datarootdir = self._datarootdir
+        prefix = self.package_folder
+        if tools.os_info.is_windows:
+            datarootdir = tools.unix_path(datarootdir)
+            prefix = tools.unix_path(prefix)
+        conf_args = [
+            "--datarootdir={}".format(datarootdir),
+            "--prefix={}".format(prefix),
+        ]
+        self._autotools.configure(args=conf_args, configure_dir=self._source_subfolder)
+        return self._autotools
+
+    def _patch_files(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+    def build(self):
+        self._patch_files()
+        autotools = self._configure_autotools()
+        autotools.make()
+
+    def package(self):
+        self.copy("COPYING*", src=self._source_subfolder, dst="licenses")
+        autotools = self._configure_autotools()
+        autotools.install()
+        tools.rmdir(os.path.join(self.package_folder, "bin", "share", "info"))
+        tools.rmdir(os.path.join(self.package_folder, "bin", "share", "man"))
+        tools.rmdir(os.path.join(self.package_folder, "bin", "share", "doc"))
+
+        if self.settings.os_build == "Windows":
+            binpath = os.path.join(self.package_folder, "bin")
+            for filename in os.listdir(binpath):
+                fullpath = os.path.join(binpath, filename)
+                if not os.path.isfile(fullpath):
+                    continue
+                os.rename(fullpath, fullpath + ".exe")
+
+    def package_id(self):
+        self.info.include_build_settings()
```

Comment:
This is only needed when you have `settings = "os", "arch", "os_build", "arch_build"`, otherwise it is not used

Reply:
I was under the impression the CI created only one package due to
https://github.com/conan-io/conan-center-index/pull/658#issuecomment-581456618
Click on *All logs here*

Reply:
There are indeed 3 different packages as expected: Windows, Linux & Mac

Reply:
https://c3i.jfrog.io/c3i/misc/logs/pr/658/10/automake/1.16.1/ shows one log for me.

Reply:
But maybe, it was a fluke?

Reply:
That one is build number 10 which is outdated (maybe the ecipe was incomplete at that commit)

- Build 12 has 3 packages: https://c3i.jfrog.io/c3i/misc/logs/pr/658/12/automake/1.16.1/
- Build 13 has 3 packages as well: https://c3i.jfrog.io/c3i/misc/logs/pr/658/13/automake/1.16.1/

Reply:
Thanks for digging into this.
I thought it was weird because that job was run automatically after some autoconf pr was merged.